python_modules/dagster/dagster_tests/core_tests/test_logging.py

```diff
@@ -364,3 +365,42 @@ def logged_solid(context):
 
     assert re.search("test OUTPUT debug logging from logged_solid.", captured.err, re.MULTILINE)
     assert re.search("test INPUT debug logging from logged_solid.", captured.err, re.MULTILINE)
+
+
+def test_conf_file_logging(capsys):
+    config_settings = {
+        "python_logs": {
+            "handlers": {
+                "handlerOne": {
+                    "class": "logging.StreamHandler",
+                    "level": "INFO",
+                    "stream": "ext://sys.stdout",
+                },
+                "handlerTwo": {
+                    "class": "logging.StreamHandler",
+                    "level": "ERROR",
+                    "stream": "ext://sys.stdout",
+                },
+            },
+        }
+    }
+
+    instance = DagsterInstance.local_temp(overrides=config_settings)
+
+    @solid
+    def log_solid(context):
+        context.log.info("Hello world")
+        context.log.error("My test error")
+
+    @pipeline
+    def log_pipeline():
+        log_solid()
+
+    execute_pipeline(log_pipeline, instance=instance)
```

Comment:
doesn't make a big difference, but I found yet another slightly better way of doing this override stuff (automatically cleans some things up):

```
from dagster.core.test_utils import instance_for_test

def test_foo():
    with instance_for_test(overrides=...) as instance:
        execute_pipeline(log_pipeline, instance=instance)
    ...
```

sorry for continually switching things up on you :P